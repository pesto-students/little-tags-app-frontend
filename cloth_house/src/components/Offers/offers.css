.carouseloffers {
  position: relative;
  height: 300px;
  line-height: 160px;
}



.carouseloffers .offertextboxright{
    position: absolute;
    top:  50%;
    right: -15%;
    transform: translate(-50%, -50%);
    height: 200px;
    border-radius: 10px;
    width: 600px;
    background-color: rgb(196 27 27 / 14%);
    backdrop-filter: blur(5px);
}

.carouseloffers .offertextboxleft{
  position: absolute;
  top:  50%;
  left: 25%;
  transform: translate(-50%, -50%);
  height: 200px;
  border-radius: 10px;
  width: 600px;
  background-color: rgb(196 27 27 / 14%);
  backdrop-filter: blur(5px);
}

.carouseloffers  h2{
  font-family: Lato;
  font-weight: bold;
  color: black;
  height: 10%;
}

.carouseloffers  h4{
  font-family: Lato;
  font-weight: bold;
  color: black;
  height: 20%;
}







.offerscards{
    padding-top: 0.5%;
    padding-left: 2%;
    padding-right: 2%;
    padding-bottom: 0.5%;
}
