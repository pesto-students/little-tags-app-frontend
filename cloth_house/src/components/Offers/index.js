import { React } from "react";
import {  Row, Col,  Carousel ,Card  } from "antd";
// import { DownOutlined } from "@ant-design/icons";
// import LangContext,{langOptions} from "../../context/LangContext.js"
// import { injectIntl } from "react-intl";
import "./offers.css";


const CarouselOffers =() =>{
    return (<>
    <Carousel autoplay>
    <div>
      <h3 className="carouseloffers">Offer 1</h3>
    </div>
    <div>
      <h3 className="carouseloffers">Offer 2</h3>
    </div>
    <div>
      <h3 className="carouseloffers">Offer 3</h3>
    </div>
    <div>
      <h3 className="carouseloffers">Offer 4</h3>
    </div>
  </Carousel>
    </>);
}



const OfferCard =() => {
    return (<>
    <Card
    hoverable
    style={{ width: 400 }}
    cover={<img alt="example" src="https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png" />}
  >
    <Card.Meta title="Bags 30% OFF" description="Shop Now" />
  </Card>,
    </>);
}

const Offers = () => {
    return (<div className="offerscards">
     <Row gutter={{ xs: 8, sm: 16, md: 24, lg: 32 }} justify="space-around" align="middle">
         <Col span={8}> 
         <OfferCard />
         </Col>
         <Col span={8}> 
         <OfferCard />
         </Col>
         <Col span={8}> 
         <OfferCard />
         </Col>
     </Row>
    </div>);
};

export {CarouselOffers,Offers};