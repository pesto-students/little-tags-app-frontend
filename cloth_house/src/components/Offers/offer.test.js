import createComponentWithIntl from '../../utils/helper_snapshot'
import Offers from "./index"

it('Agreement Modal Snapshot', () => {
    const component = createComponentWithIntl(<Offers />);
    let tree = component.toJSON()
  
    expect(tree).toMatchSnapshot()
  
  });