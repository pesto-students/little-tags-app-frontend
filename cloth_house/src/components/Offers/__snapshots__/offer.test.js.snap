// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agreement Modal Snapshot 1`] = `
<div
  className="ant-carousel"
>
  <div
    className="slick-slider slick-initialized"
    dir="ltr"
  >
    <div
      className="slick-list"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={null}
      onMouseMove={null}
      onMouseUp={[Function]}
      onTouchCancel={null}
      onTouchEnd={[Function]}
      onTouchMove={null}
      onTouchStart={[Function]}
      style={Object {}}
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "WebkitTransform": "translate3d(0px, 0px, 0px)",
            "WebkitTransition": "",
            "msTransform": "translateX(0px)",
            "opacity": 1,
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      >
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={-1}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/sunglasses.jpg"
                width="100%"
              />
              <div
                className="offertextboxleft"
              >
                <h2>
                  Offer 4 Heading
                </h2>
                <h4>
                  Offer 4 Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/yellowpic.jpg"
                width="100%"
              />
              <div
                className="offertextboxright"
              >
                <h2>
                  Offer Heading
                </h2>
                <h4>
                  Offer Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/mensuits.jpg"
                width="100%"
              />
              <div
                className="offertextboxleft"
              >
                <h2>
                  Offer 2 Heading
                </h2>
                <h4>
                  Offer 2 Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/bags.jpg"
                width="100%"
              />
              <div
                className="offertextboxleft"
              >
                <h2>
                  Offer 3 Heading
                </h2>
                <h4>
                  Offer 3 Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={3}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/sunglasses.jpg"
                width="100%"
              />
              <div
                className="offertextboxleft"
              >
                <h2>
                  Offer 4 Heading
                </h2>
                <h4>
                  Offer 4 Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={4}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/yellowpic.jpg"
                width="100%"
              />
              <div
                className="offertextboxright"
              >
                <h2>
                  Offer Heading
                </h2>
                <h4>
                  Offer Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={5}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/mensuits.jpg"
                width="100%"
              />
              <div
                className="offertextboxleft"
              >
                <h2>
                  Offer 2 Heading
                </h2>
                <h4>
                  Offer 2 Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={6}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/bags.jpg"
                width="100%"
              />
              <div
                className="offertextboxleft"
              >
                <h2>
                  Offer 3 Heading
                </h2>
                <h4>
                  Offer 3 Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={7}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="carouseloffers"
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                height="300px"
                src="./assets/images/offers/sunglasses.jpg"
                width="100%"
              />
              <div
                className="offertextboxleft"
              >
                <h2>
                  Offer 4 Heading
                </h2>
                <h4>
                  Offer 4 Sub Heading
                </h4>
                <div>
                  <button
                    className="ant-btn ant-btn-dashed"
                    onClick={[Function]}
                    type="button"
                  >
                    <span>
                      Shop Now
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <ul
      className="slick-dots slick-dots-bottom"
      onMouseEnter={null}
      onMouseLeave={null}
      onMouseOver={null}
      style={
        Object {
          "display": "block",
        }
      }
    >
      <li
        className="slick-active"
      >
        <button
          onClick={[Function]}
        >
          1
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          2
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          3
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          4
        </button>
      </li>
    </ul>
  </div>
</div>
`;
